appveyor.yml

```diff
@@ -8,7 +8,7 @@ environment:
 
 install:
   - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
-  - rustup-init.exe -y --default-host %TARGET%
+  - rustup-init.exe -y --default-host %TARGET% --profile minimal
```

Comment:
... and this?

Reply:
This skips installing rust-docs which takes minutes to installing on Windows: https://github.com/rust-lang/rustup#profiles